cms/apps/media/templates/admin/media/includes/image-editor-style.js

```diff
@@ -0,0 +1,63 @@
+var Theme = {
+    'common.bi.image': 'none',
+    'common.bisize.width': '251px',
+    'common.bisize.height': '21px',
+    'common.backgroundImage': 'none',
+    'common.backgroundColor': '#1e1e1e',
+    'common.border': '0px',
+
+    // header
+    'header.display': 'none',
+
+    // load button
+    'loadButton.display': 'none',
+
+    // download button
+    'downloadButton.display': 'none',
+
+    // main icons
+    'menu.normalIcon.path': '/static/cms/svg/image-editor/icon-b.svg',
```

Comment:
Use `{% static %}` here and for all other uses (this file is `{% include %}`'d in a template so that will work, right?).